app/components/Pill.client.tsx

```diff
@@ -0,0 +1,20 @@
+import { MouseEventHandler } from 'react';
+
+interface PillProps {
+  title: string;
+  icon?: string;
+  action?: MouseEventHandler;
+}
+const Pill: React.FC<PillProps> = ({ title, icon = '💖,#️⃣', action }) => (
+  <button className="relative rounded-full bg-slate-200 group py-2 pr-4 pl-3" onClick={action}>
+    <span className="absolute pr-2 opacity-0 group-hover:opacity-100 ease-in-out duration-300">
```

Comment:
Is it advisable to split the icons present.
We are looking for a generic Pill Component here which should handle icons.
Could we look into the way of integration of react-icons or react-feather here please.
We could pass the icon as a component here as well.
Could we look into handling this